extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -32,6 +33,7 @@ import {toggle} from '../../../src/style';
 
 installCidService(AMP.win);
 installStorageService(AMP.win);
+installActivityService(AMP.win);
```

Comment:
@cramforce I see the reason why it is better to put this here. How do you make sure that any place that is using activity service calls this first and is not just dependent on amp-analytics calling it for them? For example, how does amp-pixel work with this var? do we need a call to install the service there as well?

Would having this code in url-replacements be bad?